http/proxy_query_service.go

```diff
@@ -9,37 +9,53 @@ import (
 	"net/http"
 
 	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/iocounter"
+	icontext "github.com/influxdata/influxdb/context"
 	"github.com/influxdata/influxdb/query"
 	"github.com/julienschmidt/httprouter"
 	"github.com/prometheus/client_golang/prometheus"
 	"go.uber.org/zap"
 )
 
+// This file contains ProxyQueryHandler and ProxyQueryService
+// which implement services and handlers for both queryd and transpilerde.
```

Comment:
I don't think we should reference names for the enterprise product in open source. It's not a problem with information security, but more that the core product should be able to stand on its own as an individual piece of code in my opinion.

Reply:
This is a good point.  Actually this code is only used in the enterprise product, not in OSS at all.  A follow-up would be to move this into idpe.  I can change the comment in the meantime.